src/TraceEvent/Microsoft.Diagnostics.Tracing.TraceEvent.nuspec

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0"?>
+<package>
+  <metadata minClientVersion="2.5">
+    <id>Microsoft.Diagnostics.Tracing.TraceEvent</id>
+    <version>0.0.0</version>
+    <title>Microsoft TraceEvent Library</title>
+    <authors>Microsoft</authors>
+    <owners>Microsoft</owners>
+    <licenseUrl>http://go.microsoft.com/fwlink/?LinkId=329770</licenseUrl>
+    <iconUrl>http://go.microsoft.com/fwlink/?LinkID=288859</iconUrl>
+    <requireLicenseAcceptance>true</requireLicenseAcceptance>
+    <description>
+      Event Tracing for Windows (ETW) is a powerful logging mechanism built into the Windows OS and is used extensively in Windows. You can also log ETW events yourself code using the System.Diagnostics.Tracing.EventSource class.
+
+      The TraceEvent library conains the classes needed to control ETW providers (including EventSources) and parse the events they emit.
+
+      The library includes
+      -- TraceEventSession which can enable ETW providers,
+      -- EtwTraceEventSource which lets you read the stream of ETW events, and
+      -- TraceLog which is is digested form of ETW events which include decoded stack traces associated with the events.
+      -- A Programmers Guide for using the classes above.
+
+      Also see the TraceEvent Samples NuGet package for example uses of this library.
+    </description>
+    <summary>TraceEvent is a .NET Framework library for capturing and analyzing ETW events.</summary>
+    <releaseNotes>https://github.com/Microsoft/perfview/releases/tag/V$version$</releaseNotes>
+    <copyright>Copyright © Microsoft Corporation</copyright>
+    <tags>TraceEvent EvenSource Microsoft ETW Event Tracing for Windows</tags>
+  </metadata>
+
+  <files>
+    <!-- User information -->
+    <!--<file src="License-Stable.rtf" target="License-Stable.rtf" />-->
+    <file src="ReadMe.txt" target="ReadMe.txt" />
+    <file src="ReleaseNotes.txt" target="ReleaseNotes.txt" />
```

Comment:
Yes, you can remove these as well.   In general we want only one place for information to be updated.  Putting it in the releases page on GitHub is very reasonable.